.github/workflows/release.yml

```diff
@@ -16,9 +16,9 @@ on:
 jobs:
   prepare:
     if: github.repository_owner == 'viamrobotics'
-    runs-on: [self-hosted, x64]
+    runs-on: [buildjet-8vcpu-ubuntu-2204-arm]
```

Comment:
I stopped short of switching everything over to buildjet, but for some reason the `aarch64` build just didn't work unless _both_ `prepare` and `build_aarch64_linux` ran on an arm64 buildjet machine... I had to switch the container as well to avoid emulation. The other build jobs still run fine it seems.